### DISCUSSION TITLE
Realm is deprecated

### DISCUSSION THREAD

**Discussion by tsekiguchi** [realm/realm-js#6888] [General]
Mongo sent out an email yesterday notifying everyone using Atlas SDK's (Realm) that it will be deprecated by Sept 30, 2025.

This was a promising project. Realm was just acquired in 2019 by Mongo. What was the decision behind this? What happens to Realm in between now and next year?

**Comment by kraenhansen** [comment]
Duplicate of https://github.com/realm/realm-js/discussions/6884
